modules/styling/lib/number2rem.ts

```diff
@@ -0,0 +1,12 @@
+import {px2rem} from './px2rem';
+
+/**
+ *
+ * Function that converts a number to a string. If the value is a number, it will convert it to a `rem` string.
+ */
+export function number2rem(value: number | string | undefined): string {
+  if (typeof value === 'number') {
+    return px2rem(value);
+  }
+  return value || '';
```

Comment:
we do this check in a lot of places, so I just wrote a helper function to help with these scenarios:
```tsx
width: typeof width === 'number' ? px2rem(width) : width,
```

Reply:
Nit: `numberToRem` isn't really what's happening though, or that's not really the purpose of this abstraction. Why not make `px2rem` able to handle `string` values?